sklearn/cluster/affinity_propagation_.py

```diff
@@ -290,6 +290,25 @@ class AffinityPropagation(BaseEstimator, ClusterMixin):
     n_iter_ : int
         Number of iterations taken to converge.
 
+    Examples
+    --------
+    >>> from sklearn.cluster import AffinityPropagation
+    >>> import numpy as np
+    >>> X = np.array([[1, 2], [1, 4], [1, 0],
+    ...               [4, 2], [4, 4], [4, 0]])
+    >>> clustering = AffinityPropagation().fit(X)
+    >>> clustering
+    ... # doctest: +NORMALIZE_WHITESPACE
```

Comment:
please get rid of the ``...`` here

Reply:
Done.

Reply:
does that work? then also fix in https://github.com/scikit-learn/scikit-learn/pull/11483#issuecomment-405113228

Reply:
Can't do it there, since it'll make the line too long. But I can change the structure of the example so that somehow it works.